src/Aquifer.API/Modules/Resources/LanguageResources/LanguageResourcesEndpoints.cs

```diff
@@ -31,6 +31,10 @@ public static async Task<Results<Ok<ResourceItemsByChapterResponse>, NotFound, B
             return TypedResults.BadRequest("parentResourceIds query param must be specified");
         }
 
+        var lastChapterInBook = await dbContext.BibleBookChapters
```

Comment:
Fetching this isn't technically necessary unless we have passages returned with a different book ID than the requested `bookId` but it was simpler to just always fetch it here.  I could refactor to conditionally run this if desired.

Reply:
I would make this conditional if we can, because passages with a different start and end book id is going to be extremely rare.

Reply:
I updated to conditionally fetch it.